src/alchemlyb/estimators/ti_.py

```diff
@@ -105,3 +105,50 @@ def fit(self, dHdl):
         self.states_ = means.index.values.tolist()
 
         return self
+
+    def separate_dhdl(self):
+        """
+        For transitions with multiple lambda, the `dhdl` attribute would return
```

Comment:
Use rich reST markup:
```suggestion
        For transitions with multiple lambda, the attr:`dhdl` attribute would return
```